README.md

```diff
@@ -125,6 +125,14 @@ Then you can interact with the k8s API on the bootstrap VM e.g
 You can also see the status of the bootkube.sh script which is running via
 `journalctl -b -f -u bootkube.service`.
 
+## Login credentials
+
+Once succesfully deployed, you can access the OpenShift instance by using the following credentials:
+
+ Username | Password
+ -------- | ---------
+ kubeadmin  | See the content of the secret in `ocp/auth/kubeadmin-password` 
```

Comment:
we shouldn't document interacting with the username/password IMO or there's a risk they end up in the shell history, see https://github.com/openshift-metal3/dev-scripts/blob/master/README.md#interacting-with-the-deployed-cluster where we already document using the --config option to point to the kubeconfig file instead.

Note you can make this persistent in your environment by adding e.g `export KUBECONFIG=$HOME/dev-scripts/ocp/auth/kubeconfig` to your bashrc.

Reply:
Ok for the shell, but the idea here was to document how to access via the web interface, as I did not find any reference in the existing dev-scripts documentation, thus I thought it could have been a useful piece of information especially for first-users. I think at least a reference to something could be useful (ie https://docs.openshift.com/container-platform/4.1/web-console/web-console.html)

Reply:
Ah good point, perhaps we should have a section explaining how to access the web console, as there's currently some info about that in https://github.com/openshift-metal3/dev-scripts#interacting-with-the-deployed-cluster ?

Reply:
We could add a  "Accessing the web console" section next to the mentioned section and move the instructions there